docs/standard-input-output.adoc

```diff
@@ -0,0 +1,58 @@
+:page-title: Standard input/output
+:page-description: JUnit Jupiter extension for catchin standard input or output.
+
+The `StdIOExtension` adds a simple way to test your classes that read from the standard
+input (`System.in`) or write to the standard output (`System.out`).
+
+== How to use the extension
+
+You annotate your test class with `@ExtendWith(StdIOExtension.class)`. This lets JUnit know
+that you would like to use the special parameters provided by the extension in your tests.
+Then, when writing your tests, you can pass two arguments provided by the extension:
+
+StdIn::
+This is for catching input from the standard in. If you are using this parameter, you have to
+annotate you method with `@StdInSource` which takes a String array and provides the values that
+you would like to read instead of waiting for input on the standard input.
```

Comment:
```suggestion
This is for simulating input from the standard in. If you are using this parameter, you have to
annotate you method with `@StdInSource`. This annotation takes a String array, which holds values that
you would like to read instead of waiting for input on the standard input. Each value of the array represents one (line of) input.
```